### PR TITLE
Add new unitaryhack issues to landing page

### PR DESCRIPTION
**Pull request by co9olguy**

### CODE REVIEW

File: unitaryhack.md [hunk 1]
```diff
@@ -30,8 +30,40 @@ If you're interested in claiming a cash prize, the following issue has a $100 bo
...
+
+[Outdated interface introduction](https://github.com/PennyLaneAI/pennylane/issues/1155)
+
+[Add the XX operation](https://github.com/PennyLaneAI/pennylane/issues/1149)
+
+[Add the ZZ operation](https://github.com/PennyLaneAI/pennylane/issues/1148)
+
+[Add the CPhase operation](https://github.com/PennyLaneAI/pennylane/issues/1147)
+
+[Add the ISWAP operation(https://github.com/PennyLaneAI/pennylane/issues/1146)
```
Comment (co9olguy) [inline unitaryhack.md:57]:
```suggestion
[Add the ISWAP operation](https://github.com/PennyLaneAI/pennylane/issues/1146)
```

File: unitaryhack.md [hunk 2]
```diff
@@ -30,8 +30,40 @@ If you're interested in claiming a cash prize, the following issue has a $100 bo
...
 
 [Add search bar for navigating tutorials](https://github.com/PennyLaneAI/pennylane/issues/1296)
 
+[Create a separate requirements file for the CI](https://github.com/PennyLaneAI/pennylane/issues/1292)
+
+[Change the order of the covariance matrix and the vector of means internally in default.gaussian](https://github.com/PennyLaneAI/pennylane/issues/1257)
+
+[Two operations named Interferometer](https://github.com/PennyLaneAI/pennylane/issues/1256)
+
+[Add a custom gate application for Toffoli in default.qubit](https://github.com/PennyLaneAI/pennylane/issues/1215)
```
Comment (josh146) [inline unitaryhack.md:43]: Oh, this one has been done, the issue should have autoclosed when the linked PR was merged 🤔

Comment (josh146) [inline unitaryhack.md:43]:
```suggestion
[Two operations named Interferometer](https://github.com/PennyLaneAI/pennylane/issues/1256)
```

**Review by josh146 — APPROVED** [review]
Looks good, just a single issue which is no longer open

File: unitaryhack.md [hunk 3]
```diff
@@ -30,8 +30,40 @@ If you're interested in claiming a cash prize, the following issue has a $100 bo
...
+
```
Comment (co9olguy) [inline unitaryhack.md:44]:
```suggestion
```

**Pull request merged**